Neos.ContentRepository.Core/Classes/Projection/ProjectionsAndCatchUpHooks.php

```diff
@@ -13,7 +13,8 @@
      * @param array<class-string<ProjectionInterface<ProjectionStateInterface>>, CatchUpHookFactories> $catchUpHookFactoriesByProjectionClassName
      */
     public function __construct(
-        public Projections $projections,
+        public ContentRepositoryReadModelProjection $readModelProjection,
+        public Projections $additionalProjections,
         private array $catchUpHookFactoriesByProjectionClassName,
```

Comment:
todo discuss naming of `        public ContentRepositoryReadModelProjection $readModelProjection,` and if we like this structure here in ProjectionAndCatchupHooks

also the construction of this in the factory is a litte hacky